File: cmd/feature_experimentation/analyze/code-samples/flagship/net/SDK_V1/sample.cs
using Flagship;
var client = FlagshipBuilder.Start(
  "ENV_ID",
  "API_KEY"
);

var context = new Dictionary<string, object>();
context.Add("key", "value");
var visitor = client.NewVisitor("visitor_id", context);
await visitor.SynchronizeModifications();

var btnColorFlag = visitor.GetModification("btnColor", "red", true);
var btnSizeFlag = visitor.GetModification("btnSize", 13, true);
var showBtnFlag = visitor.GetModification("showBtn", false, true);

File: cmd/feature_experimentation/analyze/code-samples/flagship/net/SDK_V1/sample.vb
import Flagship = Flagship

Private Sub SurroundingSub()
    Dim client As [let] = FlagshipBuilder.Start("ENV_ID", "API_KEY")

    Dim context = New Dictionary(Of String, Object)()
    context.Add("key", "value")

    Dim visitor As [let] = client.NewVisitor("visitor_id", context)

    Dim btnColorFlag As [let] = visitor.GetModification("btnColor", "red", True)
    Dim btnSizeFlag As [let] = visitor.GetModification("btnSize", 13, True)
    Dim showBtnFlag As [let] = visitor.GetModification("showBtn", True, True)
End Sub